partial-update/src/main/java/com/azure/autorest/partialupdate/util/PartialUpdateHandler.java

```diff
@@ -65,13 +70,20 @@ public static String handlePartialUpdateForFile(String generatedFileContent, Str
         // 2. If it's module-info.java file, then go to handlePartialUpdateForModuleInfoFile
         if (compilationUnitForExistingFile.getModule().isPresent() &&
                 compilationUnitForGeneratedFile.getModule().isPresent()) {
-            return handlePartialUpdateForModuleInfoFile(generatedFileContent, existingFileContent);
+            return handlePartialUpdateForModuleInfoFile(compilationUnitForGeneratedFile, compilationUnitForExistingFile);
         }
 
-        // 3. If it's class or interface file, handle partial update for class or interface file
+        // 3. If it's package-info.java file, then go to handlePartialUpdateForPackageInfoFile
```

Comment:
Please update the javadoc of this method, as well as the javadoc for the `PartialUpdateHandler`.